src/qt/guiutil.h

```diff
@@ -234,7 +234,7 @@ namespace GUIUtil
     };
 
 #if defined(Q_OS_MAC)
-    // workaround for Qt OSX Bug:
+    // workaround for Qt macOS Bug:
```

Comment:
Can be fixed up as part of #13362 or not at all.

Reply:
Should be going away in #13617. I can cherry-pick some of the other changes here into that.